packages/plugins/integration-zapier/zapier/src/authentication.ts

```
@@ -1,14 +1,44 @@
+	/** Connection label to identify this account in the UI */
+	connectionLabel: async (z: ZObject, bundle: Bundle) => {
+		try {
+			const response = await z.request({
+				url: `${process.env.API_BASE_URL}/api/user/me`,
+				headers: {
+					Authorization: `Bearer ${bundle.authData.acces_token}`
+				}
+			});
+			// Format the connection label with user information
+			const userData = response.data;
+			if (userData && userData.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)